tee-worker/local-setup/launch.py

```diff
@@ -266,28 +275,38 @@ def main(processes, config_path, parachain_type, log_config_path, offset, parach
     parser = argparse.ArgumentParser(
         description="Run a setup consisting of a node and some workers"
     )
-    parser.add_argument("config", type=str, help="Config for the node and workers")
+    parser.add_argument("--config", type=str, help="Config for the node and workers")
     parser.add_argument(
-        "parachain",
+        "--parachain",
         nargs="?",
         default="local-docker",
         type=str,
         help="Config for parachain selection: local-docker / local-binary / remote",
     )
     parser.add_argument(
-        "log_config_path",
+        "--log_config_path",
         nargs="?",
         default="./local-setup/worker-log-level-config.toml",
         type=str,
         help="log level config file path"
     )
     parser.add_argument(
-        "offset", nargs="?", default="0", type=int, help="offset for port"
+        "--offset", nargs="?", default="0", type=int, help="offset for port"
     )
     parser.add_argument(
-        "parachain_dir", nargs="?", default="/tmp/parachain_dev", help="Parachain directory for local binary"
+        "--parachain_dir", nargs="?", default="/tmp/parachain_dev", help="Parachain directory for local binary"
     )
     args = parser.parse_args()
+    
+    if not is_directory_accessible(args.parachain_dir): 
+        print("Directory is not accessible, Reassigning the directory")
+        today = datetime.datetime.now()
+        formatted_date = today.strftime('%d-%m-%Y')
+        directory_name = f"parachain_dev-{formatted_date}"
```

Comment:
Maybe it's better not to mix `_` and `-` in the folder name, we could always use `_` in this case

Reply:
Updated ✅